### ISSUE TITLE
'U_SPIFFS' was not declared in this scope

### ISSUE THREAD

**Issue by pbezza** [n0bel/ESPrinkler2#9]
Error related to this line
 if (!Update.begin(fullSize, U_SPIFFS)) {

I had to change to  if (!Update.begin(fullSize, U_FS)) {

It now compiles.

**Comment by Meins321** [comment]
Does it also work after flash? Looks like you changed the filesystem as spiff is now regarded as obsolete what I read everywhere :-/

**Comment by pbezza** [comment]
Yes, works perfectly after flashing. Not sure what was different about my ide. I had the required libraries. It's been a while since I compiled this so can't remember all details. I was also mucking around with another project around this time that requires spiffs so I could have changed a setting that affected this?

**Comment by Meins321** [comment]
Does your ESP8266 keep the wifi connection? Mine disconnects from the fritzbox 7590 and the does not come back to my wifi somehow :-/

i do suspect that the powersaving/Sleep mode for the wifi is still there and if you don't access it after the powerup it will just disconnect after some more minutes even when in range

i did change over to the younger softwar here but still have this issue:
https://github.com/lawabb/ESPrinkler2
